### PR TITLE
robot id for mocap.launch

### PR DESCRIPTION
**Pull request by kanke-hayata**
I changed mocap.launch to be used for multiple robots without editing.

### CODE REVIEW

File: aerial_robot_base/launch/external_module/mocap.launch [hunk 1]
```diff
@@ -1,13 +1,19 @@
 <launch>
+
+  <arg name="robot_id" default=""/>
```
Comment (tongtybj) [inline aerial_robot_base/launch/external_module/mocap.launch:3]: @kanke-hayata 

I think these four lines are redundant, and  following one  line is enough
```
 <arg name="robot_id" default="1"/>
```

File: aerial_robot_base/launch/external_module/mocap.launch [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
     type="mocap_node"
     name="mocap_node"
     respawn="false"
     launch-prefix=""
     required="true">
-    <rosparam>
+    <rosparam subst_value="true">
       rigid_bodies:
-         '1':
+         '$(arg mocap_id)':
```
Comment (tongtybj) [inline aerial_robot_base/launch/external_module/mocap.launch:16]: change to `$(arg robot_id)`

**Comment by kanke-hayata** [comment]
In addition to changes in the mocap.launch, I expect  changes in the launch file bringing up the mocap.launch as follows.

e.g.
```
<!-- mocap -->
<include file="$(find aerial_robot_base)/launch/external_module/mocap.launch" />
```
to
```
<!-- mocap -->
<include file="$(find aerial_robot_base)/launch/external_module/mocap.launch">
  <arg name="robot_id" value="$(arg robot_id)"/>
</include>
```

In that case, the value of the "robot_id" is given to the mocap.launch even if the value is "" (null). I think it causes a bug. Is that wrong?

**Comment by chibi314** [comment]
@kanke-hayata 
I understand the problem you mentioned.
However, I prefer the @tongtybj 's opinion because
- If you use a single robot, the `robot_id` argument should not be passed to the `mocap.launch`.
- If you use two robots but temporarily use one, you should obviously define the robot id.

**Review by chibi314 — APPROVED** [review]

**Review by tongtybj — APPROVED** [review]

**Pull request merged**